code/game/objects/effects/gibs.dm

```diff
@@ -1,62 +0,0 @@
-/proc/gibs(atom/location, list/viruses, datum/dna/MobDNA)
-	new /obj/effect/gibspawner/generic(location,viruses,MobDNA)
-
-/proc/hgibs(atom/location, list/viruses, datum/dna/MobDNA)
-	new /obj/effect/gibspawner/human(location,viruses,MobDNA)
-
-/proc/xgibs(atom/location, list/viruses)
-	new /obj/effect/gibspawner/xeno(location,viruses)
-
-/proc/robogibs(atom/location, list/viruses)
-	new /obj/effect/gibspawner/robot(location,viruses)

```

Comment:
Completely useless procs, every instances of those is just replaced by "new /obj/effect/gibspawnerstuff()"